### PR TITLE
Fix condition for monitor down alerting tests

### PR DESCRIPTION
**Pull request by fbalak**
Fixes https://github.com/red-hat-storage/ocs-ci/issues/5746

Signed-off-by: Filip Balak <fbalak@redhat.com>

### CODE REVIEW

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/11915/
Test failed because of a bug but the condition passed.

**Review by OdedViner — DISMISSED** [review]

File: tests/manage/monitoring/conftest.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def stop_mon():
         if (
             not split_index == 1
             or config.ENV_DATA["platform"].lower()
-            in constants.MANAGED_SERVICE_PLATFORMS
+            not in constants.MANAGED_SERVICE_PLATFORMS
```
Comment (ebondare) [inline tests/manage/monitoring/conftest.py:191]: This change makes the first condition pointless for all non-mananged deployments.

**Review by ebondare — CHANGES_REQUESTED** [review]

File: tests/manage/monitoring/conftest.py (hunk 1 above)
Comment (fbalak) [reply tests/manage/monitoring/conftest.py:191]: Right, thanks

Comment (fbalak) [reply tests/manage/monitoring/conftest.py:191]: Updated

**Review by ebondare — APPROVED** [review]
LGTM

**Review by mbukatov — APPROVED** [review]

**Pull request merged**